sway-lib-std/src/u128.sw

```diff
@@ -28,7 +28,7 @@ pub trait From {
 impl From for U128 {
     pub fn from(upper: u64, lower: u64) -> U128 {
         U128 {
-            upper, lower,
+            upper, lower, 
```

Comment:
Did the formatter do this?

Reply:
Oh, weird. I assume so. I didn't touch anything here.

Reply:
Resolved [2812ac0](https://github.com/FuelLabs/sway/pull/2187/commits/2812ac022b478045e825643b340cdb3533fb7342)